src/components/Counter/Counter.st.css

```diff
@@ -0,0 +1,112 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: applyUnit, color, fallback, font, value;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-5;
+    DefaultDisabledColor: color-3;
+    DefaultErrorColor: value(errorColor);
+    DefaultBackgroundColor: color-1;
+    DefaultWidth: 96;
+}
+
+:vars {
+    /*Overrides*/
+    MainColor: --overridable;
+    DisabledColor: --overridable;
+    MainWidth: --overridable;
+}
+
+:vars {
+    counterColor: color(fallback(value(MainColor), value(DefaultColor)));
+    counterDisabledColor: color(fallback(value(DisabledColor), value(DefaultDisabledColor)));
+    counterWidth: applyUnit(fallback(value(MainWidth), value(DefaultWidth)), px)
+}
+
+.root {
+    -st-states: error, disabled;
+    display: inline-flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    width: value(counterWidth);
+    height: 40px;
+    padding: 0 6px;
+    background: color(value(DefaultBackgroundColor));
+    border-width: 1px;
+    border-style: solid;
+    border-color: color(value(counterColor));
+}
+
+.root:disabled {
+    border-color: color(value(counterDisabledColor));
+}
+
+.root:error {
+    border-color: value(DefaultErrorColor);
+}
+
+.btn {
+    background: none;
+    padding: 6px;
+    border: none;
+    fill: color(value(counterColor));
+    cursor: pointer;
+}
+
+.btn[disabled] {
```

Comment:
why not use ```.root:disable .btn``` instead of ```[disabled]```?

Reply:
@jonathanadler button can be disabled both by disabled property that impacts `.root` element and by itself (when `min` or `max` value is reached)

Reply:
oh, I see 👍